backend/inspirehep/curation/views.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from flask import Blueprint, abort
+from invenio_db import db
+from webargs import fields
+from webargs.flaskparser import FlaskParser
+
+from inspirehep.accounts.decorators import login_required_with_roles
+from inspirehep.accounts.roles import Roles
+from inspirehep.records.api import LiteratureRecord
+from inspirehep.serializers import jsonify
+
+blueprint = Blueprint("inspirehep_curation", __name__, url_prefix="/curation")
+parser = FlaskParser()
```

Comment:
why do we need this? `from webargs.flaskparser import parser` should be enough no?

Reply:
I am overriding parser's `handle_error` method, it won't work without that

Reply:
why do you need to override `handle_error`?

Reply:
To return custom message and status code, without that it would be `422 Unprocessable Entity` without detailed message